src/Faker/Provider/ng_NG/Address.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Faker\Provider\ng_NG;
+
+class Address extends \Faker\Provider\Address
+{
+    protected static $postcode = array('#####', '## ###');
+
+    private static $county = array(
+        'Abuja', 'Abia', 'Adamawa', 'Akwa Ibom', 'Anambra',
+        'Bauchi', 'Bayelsa', 'Benue', 'Borno',
+        'Cross River',
+        'Delta',
+        'Ebonyi', 'Edo', 'Ekiti', 'Enugu', 'Gombe',
+        'Imo', 'Jigawa',
+        'Kaduna', 'Kano', 'Katsina', 'Kebbi', 'Kogi', 'Kwara',
+        'Lagos',
+        'Nasarawa', 'Niger',
+        'Ogun', 'Ondo', 'Osun', 'Oyo',
+        'Plateau',
+        'Rivers',
+        'Sokoto',
+        'Taraba',
+        'Yobe',
+        'Zamfara'
+    );
+
+    protected static $country = array(
+        'Afghanistan', 'Albania', 'Algeria', 'American Samoa', 'Andorra', 'Angola', 'Anguilla', 'Antarctica (the territory South of 60 deg S)', 'Antigua and Barbuda', 'Argentina', 'Armenia', 'Aruba', 'Australia', 'Austria', 'Azerbaijan',
```

Comment:
You are right. However, the English names of countries are very commonly used in Nigeria. We do not really have localised names.